tests/test_registry.py

```diff
@@ -13,11 +13,17 @@
     ContainerImagePushItem,
     SourceContainerImagePushItem,
 )
+from pushsource._impl.utils.containers import (
```

Comment:
Perhaps in the context of this repository it's obvious what `MT` and `S2` mean, but for someone unfamiliar with it the names are completely opaque. Something like `MEDIATYPE_SCHEMA2_V1` would be immediately clear.